### PR TITLE
Implement VISA

### PR DESCRIPTION
**Pull request by esarver**
Implement VISA while still supporting the raw protocol over Lan sockets.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -42,7 +43,10 @@ thiserror = "1.0.58"
 tmc = { git = "https://github.com/esarver/rusb-usbtmc" }
 tracing = { version = "0.1.40", features = ["async-await"] }
 tracing-subscriber = { version = "0.3.18", features = ["json"] }
-tsp-toolkit-kic-lib = { git = "https://github.com/tektronix/tsp-toolkit-kic-lib.git", tag = "v0.17.0" }
+tsp-toolkit-kic-lib = { git = "https://github.com/tektronix/tsp-toolkit-kic-lib.git", branch = "task/implement-visa" }
+
+[workspace.features]
+visa = ["tsp-toolkit-kic-lib/visa"]
```
Comment (esarver) [inline Cargo.toml:49]:
```suggestion
```

File: kic-discover/Cargo.toml [hunk 2]
```diff
@@ -33,3 +33,5 @@ tracing = { workspace = true }
 tracing-subscriber = { workspace = true }
 tsp-toolkit-kic-lib = { workspace = true }
 
+[features]
+visa=["tsp-toolkit-kic-lib/visa"]
```
Comment (esarver) [inline kic-discover/Cargo.toml:37]:
```suggestion
```

File: kic/src/main.rs [hunk 3]
```diff
@@ -220,12 +221,33 @@ fn cmds() -> Command {
 }
 
 fn main() -> anyhow::Result<()> {
+    eprintln!("args: {:?}", std::env::args().skip(1));
```
Comment (esarver) [inline kic/src/main.rs:224]:
```suggestion
```

**Review by jharajeev55 — APPROVED** [review]

**Comment by esarver** [comment]
Forcing merge since the only updates since approval were to the `kic-lib` version

**Pull request merged**